README.md

```diff
@@ -36,6 +36,8 @@ Contact us
 ----------
 * [Mailing List](https://groups.google.com/forum/#!forum/openrefine)
 * [Twitter](http://www.twitter.com/openrefine)
+* [Matrix/Element Gitter bridge (mobile friendly)](https://matrix.to/#/#OpenRefine_OpenRefine:gitter.im)
+* [Gitter](https://gitter.im/OpenRefine/OpenRefine)
```

Comment:
```suggestion
* [Gitter](https://gitter.im/OpenRefine/OpenRefine)
* [Matrix (bridged from Gitter)](https://matrix.to/#/#OpenRefine_OpenRefine:gitter.im)
```
Let's list the native access to Gitter first, followed by the Matrix bridge. It's still accessible via the web, including mobile web browsers. It looks like Element is just one of a variety of Matrix client apps available.

Reply:
Sure, I agree Tom.